sorc/IMSaggregate_mod.f90

```diff
@@ -444,9 +478,9 @@ subroutine read_fcst(path, date_str, idim, jdim, vetfcs, swefcs, sndfcs, stcfcs,
 
         do t =1,6
             write(tt, "(i1)") t
-            fcst_file = trim(path)//trim(date_str)// & 
-                                "0000.sfc_data.tile"//tt//".nc"
-
+            fcst_file = trim(path)//trim(date_str)//"0000.sfc_data.tile"//tt//".nc"
+            print *, path
```

Comment:
Can we remove the print statements?

Reply:
yes, I am done for this. I also remove print *, date_str. Thank you.